### DISCUSSION TITLE
自动迁移时，实体字段的注释没有自动映射成数据库表字段的注释

### DISCUSSION THREAD

**Discussion by quqi-beep** [dotnetcore/FreeSql#1534] [General]
![image](https://github.com/dotnetcore/FreeSql/assets/59988881/eaa22eee-30d9-43ea-8f2b-e1a2cfef302e)
![image](https://github.com/dotnetcore/FreeSql/assets/59988881/1124724f-6e86-47bc-99d3-94c541ce722e)
![image](https://github.com/dotnetcore/FreeSql/assets/59988881/c95ba365-b14e-4d47-8b84-d00e35debc17)

请问freesql是否支持实体的xml注释自动映射为表字段的描述

**Comment by quqi-beep** [comment]
或者是否有扩展的特性添加数据库描述呢

**Reply by luoyunchong** [reply]
https://freesql.net/guide/entity-attribute.html#%E5%A4%87%E6%B3%A8

**Reply by quqi-beep** [reply]
你给的这个连接具体那个特性可以添加自定义描述呢

**Reply by luoyunchong** [reply]
![image](https://github.com/dotnetcore/FreeSql/assets/18613266/be60bab6-3fca-4adc-a68f-cf4d5a32806a)

**Reply by quqi-beep** [reply]
这个是生成xml文档的，我是想问freesql有没有支持生成或添加数据库字段注释的，比如在freesql中通过某个属性或者扩展方法设置一下，能在下图中箭头的地方显示出来
![image](https://github.com/dotnetcore/FreeSql/assets/59988881/2811556d-1f6b-473d-bc5a-ff0124fb9db7)

**Reply by densen2014** [reply]
> 这个是生成xml文档的，我是想问freesql有没有支持生成或添加数据库字段注释的，比如在freesql中通过某个属性或者扩展方法设置一下，能在下图中箭头的地方显示出来
> 
> ![image](https://github.com/dotnetcore/FreeSql/assets/59988881/2811556d-1f6b-473d-bc5a-ff0124fb9db7)
> 
> 

again！


https://freesql.net/guide/entity-attribute.html#%E5%A4%87%E6%B3%A8

**Reply by quqi-beep** [reply]
![image](https://github.com/dotnetcore/FreeSql/assets/59988881/f2f6729a-4990-4ebc-a428-60262216a3ed)
找到了，谢谢
